AntennaTracker/tracking.cpp

```diff
@@ -9,22 +9,35 @@
 void Tracker::update_vehicle_pos_estimate()
 {
     // calculate time since last actual position update
-    float dt = (AP_HAL::micros() - vehicle.last_update_us) * 1.0e-6f;
+    float dt_gps = (AP_HAL::micros() - vehicle.last_gps_update_us) * 1.0e-6f;
 
     // if less than 5 seconds since last position update estimate the position
-    if (dt < TRACKING_TIMEOUT_SEC) {
+    if (dt_gps < TRACKING_TIMEOUT_SEC) {
         // project the vehicle position to take account of lost radio packets
         vehicle.location_estimate = vehicle.location;
-        float east_offset = vehicle.vel.x * dt;
-        float north_offset = vehicle.vel.y * dt;
-        //location_update(vehicle.location_estimate, vehicle.heading, vehicle.ground_speed * dt);
+        float east_offset = vehicle.vel.x * dt_gps;
+        float north_offset = vehicle.vel.y * dt_gps;
         location_offset(vehicle.location_estimate, north_offset, east_offset);
         // set valid_location flag
         vehicle.location_valid = true;
     } else {
         // vehicle has been lost, set lost flag
         vehicle.location_valid = false;
     }
+
+    //calculate the altitude offset using baro data
+    if(g.alt_source == ALT_SOURCE_BARO){
```

Comment:
Minor formatting issue here too.  There should be a single space between the "if" and the "(".  Also another after the ")" and before the "{".
A little further down the "else" should appear immediately after the "}"